### PR TITLE
Change navbar color to peru

### PR DESCRIPTION
**Pull request by trouni**
Changes the navbar color as per the customer's request.

zoom_floating_video_window_and_RailsAuthentication

### CODE REVIEW

**Comment by dmbf29** [comment]
wow peru looks incredible

File: app/assets/stylesheets/components/_navbar.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .navbar-lewagon {
   justify-content: space-between;
-  background: white;
+  background: peru;
```
Comment (dmbf29) [inline app/assets/stylesheets/components/_navbar.scss:3]: Big fan of peru

**Review by dmbf29 — APPROVED** [review]
LGTM

File: app/assets/stylesheets/components/_navbar.scss (hunk 1 above)
Comment (trouni) [reply app/assets/stylesheets/components/_navbar.scss:3]: Fan of the country, not the color 🤮

**Pull request merged**